src/python/pants/notes/1.15.x.rst

```diff
@@ -6,30 +6,24 @@ This document describes releases leading up to the ``1.15.x`` ``stable`` series.
 1.15.0rc2 (4/02/2019)
 ---------------------
 
+As of this release, Pants running under Python 3 and Python 2 have reached implementation and
```

Comment:
Nit, should be “has”. 

Should we link to the instructions for running with Python 3? In this release, they still must use `pants_runtime_python_version` so that google groups email I sent is relevant.

Reply:
Thanks: I'll fix the grammar issue before landing.

I think that linking the instructions is best left to email. But it would be good if you could prepare an announcement that you'd like to send as part of the final `1.15.0` release (maybe later this week, or next Monday?)